notebooks/community/vertex_endpoints/optimized_tensorflow_runtime/bert_optimized_online_prediction.ipynb

```diff
@@ -242,6 +243,19 @@
         "## Before you begin"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b37054d4d064"
+      },
+      "source": [
+        "### Select a GPU runtime\n",
+        "\n",
+        "**Make sure you're running this notebook in a GPU runtime if you have that option. In Colab, select \"Runtime --> Change runtime type > GPU\"**.\n",
+        "\n",
+        "Please note that in order for you to be able to fine-tune model using GPU, your VM needs to have GPU drivers and CUDA 11.2 installed. You can use [TensorFlow Enterprise 2.7](https://cloud.google.com/tensorflow-enterprise/docs/use-with-notebooks) user-managed notebook instance or Colab with GPU runtime for this."
```

Comment:
Maybe some wording like. "This Colab has been tested and verified to work with TensorFlow Enterprise 2.7 and 2.8" is better because it helps provide the reader some idea of what we know works.

Reply:
If you look at other notebooks here it is not common to say that it was tested on this and that environment.
General guidance that notebook should work on any environment if it is properly configured.